Alignment/APEEstimation/python/SectorBuilder_Bpix_cff.py

```diff
@@ -35,11 +35,16 @@
     name = 'BpixLayer3',
     layer = [3],
 )
+BpixLayer4 = Bpix.clone(
+    name = 'BpixLayer4',
+    layer = [4],
+)
 
 BPIXLayerSeparation = cms.VPSet(
     BpixLayer1,
     BpixLayer2,
     BpixLayer3,
+    BpixLayer4,
```

Comment:
This makes the code phase-1-dependent. Is this intended?
You could consider to add another PSet with a phase-1 prefix, or something similar.